### PR TITLE
Added SetNoclip

### PR DESCRIPTION
**Pull request by terminator-97**
Added SetNoclip

### CODE REVIEW

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -1955,5 +1955,11 @@ public void ChangeEffectIntensity(EffectType type, byte intensity, float duratio
 
         /// <inheritdoc/>
         public override string ToString() => $"{Id} {Nickname} {UserId} {Role} {Team}";
+
+        /// <summary>
+        /// Sets noclip status to a specific player.
```
Comment (Marco15453) [inline Exiled.API/Features/Player.cs:1960]: Sets noclip status to a specific player. -> Sets the player's noclip status.

File: Exiled.API/Features/Player.cs [hunk 2]
```diff
@@ -1955,5 +1955,11 @@ public void ChangeEffectIntensity(EffectType type, byte intensity, float duratio
...
+        /// </summary>
+        /// <param name="status">The value to be set.</param>
```
Comment (Marco15453) [inline Exiled.API/Features/Player.cs:1962]: The value to be set -> Whether or not noclip should be activated.

File: Exiled.API/Features/Player.cs (hunk 1 above)
Comment (Marco15453) [inline Exiled.API/Features/Player.cs:1960]: Sets noclip status to a specific player. -> Sets the player's noclip status.

File: Exiled.API/Features/Player.cs (hunk 2 above)
Comment (Marco15453) [inline Exiled.API/Features/Player.cs:1962]: The value to be set -> Whether or not noclip should be activated.

**Comment by Michal78900** [comment]
What's the difference between this and already implemented `NoClipEnabled`?
https://github.com/Exiled-Team/EXILED/blob/master/Exiled.API/Features/Player.cs#L443-L451

File: Exiled.API/Features/Player.cs (hunk 2 above)
Comment (PintTheDragon) [reply Exiled.API/Features/Player.cs:1962]: Could be just Whether noclip should be activated.

**Comment by Marco15453** [comment]
> What's the difference between this and already implemented `NoClipEnabled`? https://github.com/Exiled-Team/EXILED/blob/master/Exiled.API/Features/Player.cs#L443-L451

This would be absolutely useless because it's already implemented.

File: Exiled.API/Features/Player.cs (hunk 2 above)
Comment (Marco15453) [reply Exiled.API/Features/Player.cs:1962]: ^^

**Comment by joker-119** [comment]
This is already implemented.

**Pull request closed without merging**